File: old/Next Round/1.java
import java.util.Scanner;
public class nextround{
	public static void main(String[] arg){
		Scanner in = new Scanner(System.in);
		int limit,Smax,sum,i,j;
		int[] score = new int[100];
		i =0;
		sum = 0;
		j = 0;
		limit = in.nextInt();
		Smax = in.nextInt();
		while (limit>0){
			score[i] = in.nextInt();
			i++;
			limit--;
		}
		for(j=0;j<i;j++){
			if(score[j]>0 && score[j]>=score[Smax-1]){
				sum++;
			}
		}
		System.out.format("%d",sum);
	}
}

File: old/Next Round/1.cpp
#include <stdio.h>
int main(){
    int limit,Smax,score[100],sum,i,j;
	sum = 0;
	scanf ("%d",&limit);
	scanf("%d",&Smax);
	i =0;
	while (limit>0){
		scanf("%d",&score[i]);
		limit--;
		i++;
	}
	for (j=0;j<i;j++){
		if(score[j]>0 && score[j]>=score[Smax-1]){
			sum++;
		}
	}
	printf("%d",sum);
			
}
